content/blog/10-reasons-why-you-should-be-using-surveys.md

```diff
@@ -8,7 +8,7 @@ author: Mriganka Chandra
 createdAt: 2023-01-20
 coverImg: /img/10-reasons-why-you-are-still-not-doing-surveys.png
 coverImgAlt: Ten reasons why you are still not doing surveys
-images: [
+selectedImages: [
```

Comment:
can we please rename this to `metaImages` at all places for consistency.

Reply:
Fixed